src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,149 @@
+package duke;
+
+import duke.data.Storage;
+import duke.data.TaskList;
+import duke.io.Command;
+import duke.io.Parser;
+import duke.io.Ui;
+import duke.tasks.Deadline;
+import duke.tasks.Event;
+import duke.tasks.Task;
+import duke.tasks.Todo;
+
+import java.io.IOException;
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
+import java.util.Scanner;
```

Comment:
I think according to the coding standards, the Java imports are below the Duke imports.